smithy-aws-apigateway-traits/src/main/java/software/amazon/smithy/aws/apigateway/traits/AuthorizersTraitValidator.java

```diff
@@ -68,4 +89,32 @@ private Optional<ValidationEvent> validateService(Model model, ServiceShape serv
                 ValidationUtils.tickedList(authSchemes),
                 invalidMappings)));
     }
+
+    /**
+     * Each authorizer with the enableSimpleResponses member defined
+     * should have the authorizedPayloadFormatVersion member set to 2.0.
+     */
+    private Optional<ValidationEvent> validateEnableSimpleResponsesConfig(Map<String, AuthorizerDefinition> authorizers,
+                                                                          ServiceShape service) {
+        String invalidConfigs = authorizers.entrySet().stream()
+                .filter(entry -> entry.getValue().getEnableSimpleResponses().isPresent())
+                .filter(entry -> entry.getValue().getAuthorizerPayloadFormatVersion().isPresent())
+                .filter(entry -> !entry.getValue().getAuthorizerPayloadFormatVersion().get().equals("2.0"))
+                .map(Map.Entry::getKey)
+                .sorted()
+                .collect(Collectors.joining(", "));
+
+        if (invalidConfigs.isEmpty()) {
+            return Optional.empty();
+        }
+
+        AuthorizersTrait authorizersTrait = service.getTrait(AuthorizersTrait.class).get();
+        return Optional.of(error(service, authorizersTrait, String.format(
+                "The enableSimpleResponses member of %s is only supported when authorizedPayloadFormatVersion "
+                + "is 2.0. The following authorizers are misconfigured: %s",
+                AuthorizersTrait.ID,
+                invalidConfigs
+        )));
+    }
```

Comment:
I'm not sure if this validator is being overzealous or not.

Reply:
This isn't being overzealous. We do similar validation to assure compatible properties are set for other traits. The [HttpApiKeyAuthTraitValidator](https://github.com/awslabs/smithy/blob/main/smithy-model/src/main/java/software/amazon/smithy/model/validation/validators/HttpApiKeyAuthTraitValidator.java) does something similar for the `@HttpApiKeyAuthTrait`'s `in` and `scheme` properties.

Reply:
This isn't being overzealous. We do similar validation to assure compatible properties are set for other traits. The [HttpApiKeyAuthTraitValidator](https://github.com/awslabs/smithy/blob/main/smithy-model/src/main/java/software/amazon/smithy/model/validation/validators/HttpApiKeyAuthTraitValidator.java) does something similar for the `@HttpApiKeyAuthTrait`'s `in` and `scheme` properties.

Reply:
Great (: